tools/mqppep/mqppep_anova.xml

```diff
@@ -0,0 +1,230 @@
+<tool id="mqppep_anova" name="MaxQuant Phosphopeptide ANOVA" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" python_template_version="3.5">
+    <description>Perform ANOVA on merged and filtered data from phospho-peptide enrichment/MaxQuant pipeline</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"/>
+    <!--
+      The weird invocation used here is because knitr and install_tinytex
+      both need access to a writeable directory, but most directories in a
+      biocontainer are read-only, so this builds a pseudo-home under /tmp
+    -->
+    <command detect_errors="exit_code"><![CDATA[
+      export OLD_PWD=\$(dirname \$(pwd));
+      export HOME=/tmp\${OLD_PWD};
```

Comment:
why do you hardcode the HOME to tmp? A hardoceded /tmp is not a good idea,  you do not know if this is big, fast or exists at all during execution.

Please add a profile="21.05" to the <tool> tag und Galaxy will set a HOME for you.

Reply:
Thank you for your feedback. @bgruening

I will try it and see if it helps. The `knitR` package upon which I am relying needs to work on a file in a writable temporary directory.  What can I use as a reliable (and sufficiently high performance) alternative to `/tmp` ?

Reply:
@bgruening Ah, I have discovered `_GALAXY_JOB_TMP_DIR` and `_GALAXY_JOB_HOME_DIR` . . .

Reply:
@bgruening Indeed, those paths worked. Please review.